### PR TITLE
Running relayer: documentation

### PR DESCRIPTION
**Pull request by svyatonik**
[rendered](https://github.com/paritytech/parity-bridges-common/blob/sv-running-relayer-md/docs/running-relayer.md)

A document on how to start your own relayer. I've been trying to keep it short, but as always, there's a lot to tell.

### CODE REVIEW

File: docs/running-relayer.md [hunk 1]
```diff
@@ -0,0 +1,345 @@
...
+of details behind that simple phrase - you could find more info in the
+[High-Level Bridge Overview](./high-level-overview.md) document.
+
+Reward that is paid to relayer has two parts. The first part static and is controlled by the governance.
+It is rather small initially - e.g. you need to deliver `10_000` Kusama -> Polkadot messages to gain single
+KSM token.
+
+The other reward part is dynamic. So to deliver an XCM message from one BridgeHub to another, we'll need to
+deliver several "things" from one chain to another (omitting details: this "thing" is an XCM message). Those
+"things" are submitted as transactions and every transaction has its cost, which is:
```
Comment (serban300) [inline docs/running-relayer.md:32]: Nit: To me it seems a bit hard to follow. What do you mean by `we'll need to
deliver several "things" from one chain to another` ?

**Review by serban300 — COMMENTED** [review]
Looks good ! Just left a couple of nits.

File: docs/running-relayer.md (hunk 1 above)
Comment (svyatonik) [reply docs/running-relayer.md]: The idea was to avoid unrelated details in this document - i.e. no mentions of storage and finality proofs, delivery and confirmation transactions, ... For that there's another document. But I agree - "things" aren't much better. I've removed this "things" "thing" and only left transactions

Comment (serban300) [reply docs/running-relayer.md]: Looks good ! The only thing left is that `deliver submit` seems a bit redundant.

**Review by serban300 — APPROVED** [review]

File: docs/running-relayer.md [hunk 2]
```diff
@@ -0,0 +1,344 @@
...
+transaction that delivers the same message `100` won't be **useful**. Hence, no compensation to relayer that
+has submitted that second transaction.
+
+But what if there are several relayers running? They are noticing the same queued message `100` and
+simultaneously submit identical message delivery transactions. You may expect that there'll be one lucky
+relayer, whose transaction would win the "race" and which will receive the compensation and reward. And
+there'll be several other relayers, losing some funds on their unuseful transactions.
+
+But actually, we have a solution that invalidates transactions of "unlucky" relayers before they are
+included into the block. So at least you may sure that you won't waste your tokens on duplicate transactions.
```
Comment (acatangiu) [inline docs/running-relayer.md:58]:
```suggestion
included into the block. So at least you may be sure that you won't waste your funds on duplicate transactions.
```

File: docs/running-relayer.md [hunk 3]
```diff
@@ -0,0 +1,344 @@
...
+In exchange, such relayers may be slashed if they submit **invalid** or **non-minimal** transaction.
+
+Transactions, signed by relayers **without** active registration, on the other hand, receive no priority
+boost. It means that if there is active registered relayer, most likely all transactions from unregistered
+will be counted as **unuseful**, not included into the block and unregistered relayer won't get any reward
+for his operations.
+
+Before registering, you should know several things about your funds:
+
+- to register, you need to hold significant amount of tokens on your relayer account. As of now, it is
```
Comment (acatangiu) [inline docs/running-relayer.md:129]:
```suggestion
- to register, you need to hold significant amount of funds on your relayer account. As of now, it is
```

File: docs/running-relayer.md [hunk 4]
```diff
@@ -0,0 +1,344 @@
...
+
+To deregister, submit the simple `deregister` extrinsic when registration is expired:
+
+![Deregister Extrinsic](./bridge-relayers-deregister.png)
+
+At any time, you can prolong your registration by calling the `register` with the larger `validTill`.
+
+## Costs
+
+Your relayer account (on both Bridge Hubs) must hold enough tokens to be able to pay costs of bridge
```
Comment (acatangiu) [inline docs/running-relayer.md:179]:
```suggestion
Your relayer account (on both Bridge Hubs) must hold enough funds to be able to pay costs of bridge
```

File: docs/running-relayer.md [hunk 5]
```diff
@@ -0,0 +1,344 @@
...
+    $BRIDGE_HUB_WESTEND_IS_SECURE \
+    --bridge-hub-westend-version-mode Auto \
+    --bridge-hub-westend-signer-file /bhw.key \
+    --bridge-hub-westend-transactions-mortality 16 \
+    --lane 00000002
+```
+
+### Starting your Polkadot <> Kusama Relayer
+
+*Work in progress, get back soon*
```
Comment (acatangiu) [inline docs/running-relayer.md:314]:
```suggestion
*Work in progress, coming soon*
```

File: docs/running-relayer.md [hunk 6]
```diff
@@ -0,0 +1,344 @@
...
+You can find more info on configuring Prometheus and Grafana in the
+[Monitor your node](https://wiki.polkadot.network/docs/maintain-guides-how-to-monitor-your-node)
+guide from Polkadot wiki.
+
+We have our own set of Grafana dashboards and alerts. You may use them for inspiration.
+Please find them in this folder:
+
+- for Rococo <> Westend bridge: [rococo-westend](https://github.com/paritytech/parity-bridges-common/tree/master/deployments/bridges/rococo-westend).
+
+- for Polkadot <> Kusama bridge: *work in progress, get back soon*
```
Comment (acatangiu) [inline docs/running-relayer.md:344]:
```suggestion
- for Polkadot <> Kusama bridge: *work in progress, coming soon*
```

**Review by acatangiu — APPROVED** [review]
Really great doc! 💯
